pioinstaller/pack/packer.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2014-present PlatformIO <contact@platformio.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import io
+import os
+import pathlib
+import re
+import subprocess
+import tempfile
+import zipfile
+
+from pioinstaller.exception import InvalidFileFormat
+
+PACK_ROOT = os.path.abspath(os.path.dirname(__file__))
+PROJECT_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(PACK_ROOT)))
+
+
+def create_wheels(package_dir: str, dest_dir: str):
```

Comment:
Please remove the annotation, our code is fully compatible between Python2/3
```
tox -e py27
tox -e py37
```

Please re-test.